### ISSUE TITLE
Example of login Google 

### ISSUE THREAD

**Issue by FrancescoMiconi** [titarenko/OAuth2#120]
Where can I find the example? Thanks

**Comment by segurofacildeveloper** [comment]
there's some example in old versions... but new version based on .netcore not have any examples... 
old version also has a implemented version that you can configure all provider in web.config file...

**Comment by moh-hassan** [comment]
Can you provide us with a simple getting start guide in the readme page?

**Comment by niemyjski** [comment]
Yeah, I'm really sorry about that. I broke quite a bit and don't have time to update the docs. I can point you to code that is using google :) https://github.com/exceptionless/Exceptionless/blob/master/src/Exceptionless.Web/Controllers/AuthController.cs#L290

**Comment by KiransHub** [comment]
Still and example would be appreciated.

**Comment by niemyjski** [comment]
Would you mind creating one and submitting a pr. I've linked to a real life example

**Comment by niemyjski** [comment]
We know documentation has been lacking — sorry about that. For now, here's a real-world example of using this library with Google OAuth2 in an ASP.NET Core app:

**Exceptionless AuthController** (uses Google, GitHub, Facebook, and Microsoft clients):
https://github.com/exceptionless/Exceptionless/blob/master/src/Exceptionless.Web/Controllers/AuthController.cs

The basic pattern is:

```csharp
// 1. Create client with configuration
var config = new ClientConfiguration {
    ClientId = "your-client-id",
    ClientSecret = "your-client-secret",
    RedirectUri = "https://yourapp.com/callback",
    Scope = "openid email profile"
};
var client = new GoogleClient(new RequestFactory(), config);

// 2. Get login URL and redirect user
var loginUrl = await client.GetLoginLinkUriAsync(state: "optional-state");

// 3. In callback, get user info
var userInfo = await client.GetUserInfoAsync(Request.QueryString);
// userInfo.Email, userInfo.FirstName, userInfo.LastName, etc.
```

We'd love a community contribution to add proper documentation to the README. If anyone would like to submit a PR with a getting started guide, that would be very welcome!
